modules/openapi-generator/src/main/resources/scala-akka-http-server/controller.mustache

```diff
@@ -6,11 +6,11 @@ import {{akkaImportGroupId}}.http.scaladsl.server.Route
 {{/operations}}{{/apis}}{{/apiInfo}}
 import {{akkaImportGroupId}}.http.scaladsl.server.Directives._
 import {{akkaImportGroupId}}.actor.ActorSystem
-import {{akkaImportGroupId}}.stream.ActorMaterializer
+import {{akkaImportGroupId}}.stream.Materializer
```

Comment:
`ActorMaterializer` is deprecated

https://github.com/akka/akka/blob/4ab9d55102a1c9038950e1034ab3d34103e4a3dd/akka-stream/src/main/scala/akka/stream/ActorMaterializer.scala#L208-L209

https://github.com/apache/pekko/blob/ccb24f53b9f3c2bde48c2830fc18b4ff4a16f7c9/stream/src/main/scala/org/apache/pekko/stream/ActorMaterializer.scala#L222-L223